app/src/main/java/com/example/myfridgehome/dto/Recipe.kt

```diff
@@ -1,5 +1,7 @@
 package com.example.myfridgehome.dto
 
-data class Recipe(var getTitle : String) {
-    var title: String = getTitle
+data class Recipe(var title : String = "") {
+    override fun toString(): String {
+        return "$title"
+    }
 }
```

Comment:
Return the value of the dto instead of placing that value in a variable and returning nothing.

Reply:
This looks a bit more useful.  Also, title is a better name for a parameter than getTitle.  Variables should be attributes, not verbs.